index.html

```diff
@@ -0,0 +1,250 @@
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <div class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="content">
+                    <div class="topic">
+                        Multi-Post
+                        <br />
+                        Stories
+                    </div>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <div class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+                <img class="img" src="image/card-4.svg" alt="" />
+            </div>
+        </div>
+    </div>
+
+    <div id="about" class="myself">
+        <div class="left">
+            <div class="description">
+                <h2 class="topic">About <br /> Myself</h2>
+                Hello I'm a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If
+                you like what you see and have a project you need coded, don't
+                hestiate to contact me.
+            </div>
+            <div class="menuSection">
+                <a href="#" target="_blank">LET'S CONNECT</a>
+                <div class="socialMedia">
+                    <img src="image/Twitter.png" alt="" />
+                    <img src="image/Linkedin.png" alt="" />
+                    <img src="image/Meduim.png" alt="" />
+                    <img src="image/Github.png" alt="" />
+                    <img src="image/angellist.png" alt="" />
+                </div>
```

Comment:
- [ ]  For a good user experience, I would recommend you wrap your list of icons in a list(`<ul>`) with each icon wrapped in a  `<li>` tag because you have a list of icons and it will be semantic to have them in a list 😄.

Also, wrap your `img` tag in an anchor `<a>` tag as it is a social link that leads to the social media page when clicked 👍🏼 .

e.g. 
 ```
<ul>
  <li class="social">
       <a href="">
           <img src="image/Github.png" alt="">
        </a>
  </li>
 <li class="social">
       <a href="">
           <img src="image/angellist.png" alt="">
        </a>
  </li>
</ul>